src/create-element.js

```diff
@@ -64,6 +64,7 @@ export function createVNode(type, props, key, ref) {
 		_dom: null,
 		_lastDomChild: null,
 		_component: null,
+		_renderCallbacks: null,
```

Comment:
FYI, initializing this to an array saves us about 11 B. However it does add an extra array allocation for every VNode :/ Unsure how that affects performance. Perhaps someone can investigate this and open a new PR if its worth it?